### PR TITLE
1.14.3 release documentation

### PR DESCRIPTION
**Pull request by rnicoll**
* Refresh release notes based on Dogecoin 1.14.2 to 1.14.3 differences. Note that Dogecoin 1.14.2 was based on Bitcoin Core 0.14.3, so there's only backported changes that came from Bitcoin Core.
* Refresh the `man` pages using `help2man`

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -42,410 +42,94 @@ frequently tested on them.
...
-  converted to the new format which is not readable by prior versions of the
-  software.
-
-- Support for "priority" (coin age) transaction sorting for mining is
-  considered deprecated in Core and will be removed in the next major version.
-  This is not to be confused with the `prioritisetransaction` RPC which will remain
-  supported by Core for adding fee deltas to transactions.
-
-P2P connection management
+Locale independent sorting
```
Comment (patricklodder) [inline doc/release-notes.md:78]: I'd move this to minor

File: doc/release-notes.md [hunk 2]
```diff
@@ -42,410 +42,94 @@ frequently tested on them.
...
+    
+Some locales sort differently, which causes a different order of files, which leads to an unreproducible build.  Sorting is now always based on the C locale.
+
+Add query options to listunspent RPC call
+-----------------------------------------
+
+* Return unspents greater or equal than a specific amount in BTC: minimumAmount (default = 0).
+* Return unspents lower or equal than a specific amount in BTC: maximumAmount (default = MAX_MONEY = 21000000.0 BTC).
+* Return unspents with a total number lower or equal than a specific number: maximumCount (default=0=unlimited).
+* Return unspents which total is greater or equal than a specific amount in BTC: minimumSumAmount (default = MAX_MONEY = 21000000.0 BTC).
```
Comment (patricklodder) [inline doc/release-notes.md:89]: this is a BTC description

File: doc/man/dogecoin-cli.1 [hunk 3]
```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
```
Comment (patricklodder) [inline doc/man/dogecoin-cli.1:1]: missing cli documentation for the `listunspent` enhancements?

File: doc/release-notes.md [hunk 4]
```diff
@@ -466,14 +150,16 @@ Thanks to everyone who directly contributed to this release:
...
-- Russell Yanofsky
-- Shigeya Suzuki
+- Shibe
+- Sjors Provoost
+- Suhas Daftuar
+- Thomas Snider
+- tnaka
+- Tjps
+- Vertian
 - Wladimir J. van der Laan
```
Comment (patricklodder) [inline doc/release-notes.md:163]: I only have the following contributors when iterating the commits and removing reversals:

```
- Anthony Chen 
- BT 
- Bertrand Jacquin 
- Daniel Edgecumbe 
- Dennis Field 
- Ikko Ashimine 
- John-Gee 
- Jonathan 
- Kent 
- Luis-Johannes Schubert 
- Marco 
- Max K 
- Patrick Lodder 
- Primo 
- Reiner Herrmann 
- Ross Nicoll 
- Shibe 
- Vertian 
- marcuswin 
```

File: doc/release-notes.md [hunk 5]
```diff
@@ -42,410 +42,94 @@ frequently tested on them.
...
+* Change references in INSTALL.md from Bitcoin to Dogecoin
+* Update package links for OSX cross compilation
+* Update Patrick Lodder's private key
+* Numerous fixes to automated tests.
+* Replace missing Windows build notes.
+* Change IPC prefix from `bitcoin:` to `dogecoin:`.
+* Corrections to Italian translation.
+* Refresh main and test network checkpoints.
+* Update build-osx.md for Mac install.
+* Do not print an error on connection timeouts through proxy.
```
Comment (patricklodder) [inline doc/release-notes.md:103]: Would change this to:

```
Minor changes
=============

 * Update package links for OSX cross compilation
 * Change IPC prefix from `bitcoin:` to `dogecoin:`.
 * Locale independent sorting.
 * Corrections to Italian translation.
 * Refresh main and test network checkpoints and seeds.
 * Do not print an error on connection timeouts through proxy.
 * Numerous fixes to automated tests.
 * Numerous fixes to documentation.
 ```

File: doc/release-notes.md [hunk 6]
```diff
@@ -42,439 +42,90 @@ frequently tested on them.
...
+    
+Some locales sort differently, which causes a different order of files, which leads to an unreproducible build.  Sorting is now always based on the C locale.
+
+Add query options to listunspent RPC call
+-----------------------------------------
+
+* Return unspents greater or equal than a specific amount in BTC: minimumAmount (default = 0).
+* Return unspents lower or equal than a specific amount in BTC: maximumAmount (default = MAX_MONEY = 21000000.0 BTC).
+* Return unspents with a total number lower or equal than a specific number: maximumCount (default=0=unlimited).
+* Return unspents which total is greater or equal than a specific amount in BTC: minimumSumAmount (default = MAX_MONEY = 21000000.0 BTC).
```
Comment (rnicoll) [inline doc/release-notes.md:89]:
```suggestion
* Return unspents greater or equal than a specific amount in DOGE: minimumAmount (default = 0).
* Return unspents lower or equal than a specific amount in DOGE: maximumAmount (default = MAX_MONEY = 100000000000 DOGE).
* Return unspents with a total number lower or equal than a specific number: maximumCount (default=0=unlimited).
* Return unspents which total is greater or equal than a specific amount in DOGE: minimumSumAmount (default = MAX_MONEY = 100000000000 DOGE).
```

File: doc/release-notes.md [hunk 7]
```diff
@@ -42,439 +42,90 @@ frequently tested on them.
...
+* Change references in INSTALL.md from Bitcoin to Dogecoin
+* Update package links for OSX cross compilation
+* Update Patrick Lodder's private key
+* Numerous fixes to automated tests.
+* Replace missing Windows build notes.
+* Change IPC prefix from `bitcoin:` to `dogecoin:`.
+* Corrections to Italian translation.
+* Refresh main and test network checkpoints.
+* Update build-osx.md for Mac install.
+* Do not print an error on connection timeouts through proxy.
```
Comment (rnicoll) [inline doc/release-notes.md:103]:
```suggestion
Minor changes
=============

 * Update package links for OSX cross compilation
 * Change IPC prefix from `bitcoin:` to `dogecoin:`.
 * Locale independent sorting.
 * Corrections to Italian translation.
 * Refresh main and test network checkpoints and seeds.
 * Do not print an error on connection timeouts through proxy.
 * Numerous fixes to automated tests.
 * Numerous fixes to documentation.
```

File: doc/man/dogecoin-cli.1 (hunk 3 above)
Comment (rnicoll) [reply doc/man/dogecoin-cli.1:1]: Ah probably my binaries are out of date

Comment (rnicoll) [reply doc/man/dogecoin-cli.1:1]: Nope, the cli docs doesn't cover RPC commands it turns out

**Review by patricklodder — APPROVED** [review]

**Pull request merged**